sdk/framework/backend.go

```diff
@@ -745,6 +745,8 @@ func (t FieldType) Zero() interface{} {
 		return ""
 	case TypeInt:
 		return 0
+	case TypeInt64:
```

Comment:
It would be great to add a test to help catch any future errors like this. I didn't want to overcomplicate by getting into reflection etc, but had a little play and came up with this:

```go
func TestFieldTypeMethods(t *testing.T) {
	unknownFormat := convertType(TypeInvalid).format

	for i := TypeInvalid + 1; i < typeInvalidMax; i++ {
		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
			if i.String() == TypeInvalid.String() {
				t.Errorf("unknown type string for %d", i)
			}

			if convertType(i).format == unknownFormat {
				t.Errorf("unknown schema for %d", i)
			}

			_ = i.Zero()
		})
	}
}
```

You'll notice, that required adding a new (unexported) type to the end of the enum:

```go
	// DO NOT USE. Any new values must be inserted before this value.
	// Used to write tests that ensure type methods handle all possible values.
	typeInvalidMax
```

That turned up that the `String()` method is also missing a case for `Int64`:

```go
	case TypeInt64:
		return "int64"
```